test/helpers/setup.js

```diff
@@ -1,37 +1,32 @@
-import DestinationRequest from 'testcafe-hammerhead/lib/request-pipeline/destination-request'
 import { createTables } from '@pubsweet/db-manager'
 import config from 'config'
-import start from 'pubsweet/src/startup/start'
 import startS3Server from '@elifesciences/component-service-s3/mock'
+import db from 'pubsweet-server/src/db'
+
 // while writing tests, using the following to start the server avoids having to recompile the app
 // import {startServer: start} from 'pubsweet-server'
 
 import replaySetup from './replay-setup'
 
-let server
 let s3rver
-let serverStartAttempt = false
-
-export async function startServer() {
-  console.log('test/helpers/setup.js::startServer')
-  if (!server) {
-    if (serverStartAttempt) {
-      throw new Error(
-        "Attempting to start the server again, but a previous attempt hasn't been executed yet",
-      )
-    }
-    serverStartAttempt = true
-    // increase timeout to wait for webpack compilation
-    DestinationRequest.TIMEOUT = 180 * 1000
-    console.log('test/helpers/setup.js::startServer await start')
-    server = await start()
-    console.log('test/helpers/setup.js::startServer started')
-  }
-}
 
 export async function setup(t) {
   console.log('test/helpers/setup.js::setup')
 
+  console.log('test/helpers/setup.js::setup truncate query')
+
+  const { rows } = await db.raw(`
+    SELECT tablename
+    FROM pg_tables
+    WHERE schemaname = current_schema
+  `)
+
+  const truncateQuery = rows
+    .map(row => `TRUNCATE "${row.tablename}" CASCADE`)
+    .join(';')
+
+  await db.raw(truncateQuery)
+
   console.log('test/helpers/setup.js::setup createTables')
```

Comment:
If the tables are truncated then we don't need to recreate them? Or does this part also insert fixture data?

Reply:
In Jenkins and when running tests the tables should already be created at the point of entry. So yes, lets remove line 31 `await createTables(true)`